test/Api.Tests/ControllerTests/UsersControllerTest.cs

```diff
@@ -1,70 +1,159 @@
-﻿using System.Security.Claims;
-using PaderbornUniversity.SILab.Hip.CmsApi.Controllers;
+﻿using PaderbornUniversity.SILab.Hip.CmsApi.Controllers;
 using PaderbornUniversity.SILab.Hip.CmsApi.Models;
 using PaderbornUniversity.SILab.Hip.CmsApi.Models.Entity;
 using MyTested.AspNetCore.Mvc;
 using Xunit;
+using System.Linq;
+using PaderbornUniversity.SILab.Hip.CmsApi.Utility;
+using PaderbornUniversity.SILab.Hip.CmsApi.Models.User;
 
 namespace PaderbornUniversity.SILab.Hip.CmsApi.Tests.ControllerTests
 {
     public class UsersControllerTest
     {
+        private ControllerTester<UsersController> _tester;
+        public InviteFormModel InviteFormModel { get; set; }
+
+        public UsersControllerTest()
+        {
+            _tester = new ControllerTester<UsersController>();
+            InviteFormModel = new InviteFormModel()
+            {
+                Emails = new string[]
+                        {
+                            "abc@xyz.com", "lmn@xyz.com"
+                        }
+            };
+        }
+
+        #region InviteUsers
+
+        /// <summary>
+        /// Should return code 202 when an admin is trying to invite new users with new Emails
+        /// </summary>
+        //[Fact] - Dependency Injection is not working with XUnit and the issue is still open
+        // Refer: https://github.com/xunit/xunit/issues/687
+        public void InviteUsersTest202()
+        {
+            _tester.TestController()
+                .Calling(c => c.InviteUsers(InviteFormModel, null))
+                .ShouldHave()
+                .DbContext(db => db.WithSet<User>(newuser => newuser.Any(actual => actual.Equals(InviteFormModel))))
+                .AndAlso()
+                .ShouldReturn()
+                .StatusCode(202);
+        }
+
         /// <summary>
-        /// Tests response from Get action of UsersController.
+        /// Should return code 409 when an admin is trying to invite existing users
         /// </summary>
         [Fact]
-        public void GetUserListTest()
+        public void InviteUsersTest409()
         {
-            MyMvc
-                .Controller<UsersController>()
-                .Calling(c => c.Get(null, null, 0, 0))
+            var _student1 = new User
+            {
+                Id = 4,
+                Email = InviteFormModel.Emails[0],
+                Role = "Student"
+            };
+
+            var _student2 = new User
+            {
+                Id = 5,
+                Email = InviteFormModel.Emails[1],
+                Role = "Student"
+            };
+
+            _tester.TestController()
+                .WithDbContext(dbContext => dbContext.WithSet<User>(o => o.AddRange(_student1, _student2)))
```

Comment:
Just do this test with the already existing `_tester.Student` user, should be the same but is less complicated